messaging/MessagingExampleSwift/AppDelegate.swift

```diff
@@ -91,6 +91,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // Print full message.
     print(userInfo)
+
+    if completionHandler {
```

Comment:
Does this work? Completion handler doesn't look nullable, and I've generally had to do != nil for these tests.